Lib/test/test_logging.py

```diff
@@ -2125,6 +2125,10 @@ def test_warnings_no_handlers(self):
 def formatFunc(format, datefmt=None):
     return logging.Formatter(format, datefmt)
 
+class myCustomFormatter:
```

Comment:
See comment above about formatter classes that don't inherit from `logging.Formatter`.